app/services/charges/charge_models/package_service.rb

```diff
@@ -21,12 +21,20 @@ def unit_amount
       end
 
       def amount_details
-        return { free_units: 0, paid_units: 0, per_package_size: 0, per_package_unit_amount: 0 } if units.zero?
-        return { free_units:, paid_units: 0, per_package_size:, per_package_unit_amount: } if paid_units.negative?
+        return { free_units: '0.0', paid_units: '0.0', per_package_size: 0, per_package_unit_amount: '0.0' } if units.zero?
```

Comment:
Style/IfUnlessModifier: Modifier form of `if` makes the line too long. (https://rubystyle.guide#if-as-a-modifier)